src/lib/lines/TodayLine.js

```diff
@@ -12,10 +12,26 @@ export default class TodayLine extends Component {
   }
 
   static defaultProps = {}
+  
+  state = {
+    currentTime: new Date(),
+  }
+
+  componentDidMount(){
+    this.interval = setInterval(()=> {
```

Comment:
Are these changes specific to optimizations?  If not, can you please remove these changes.  I'd like to keep PRs focused in their intent.  

I see where you're going here and I agree with these changes but they should be in a separate PR.

Reply:
Well, this component as well had a lot of extra renders where it got rendered every time out timeline rendered which is not optimal. When I changed it to a pure component this resulted in it being never updated as expected since no state nor props where updated although time changed and it should've moved so it was more of solving a bug. Does this change worth being in a separate pull request?